proto/src/determined/trial/v1/trial.proto

```diff
@@ -59,12 +44,10 @@ message MetricsWorkload {
   determined.experiment.v1.State state = 3;
   // Metrics.
   google.protobuf.Struct metrics = 4;
-  // Number of inputs processed.
+  // DEPRECATED: Number of inputs processed.
   int32 num_inputs = 5;
-  // Number of batches in this workload.
-  int32 num_batches = 6;
-  // Number of batches processed prior to this workload.
-  int32 prior_batches_processed = 7;
+  // Total number of batches as of this workload's completion.
+  int32 total_batches = 6;
```

Comment:
should we add the deprecated option as well for num_inputs https://developers.google.com/protocol-buffers/docs/proto3#options 

also, this probably won't have any effet since we're not directly using protobuf but, let's avoid reusing the old field numbers?